ouroboros/chainsync/byronV6.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 SundaeSwap Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Byron block code for Ogmios v6.
+// https://github.com/CardanoSolutions/ogmios/blob/d2b1d70ab5e676b5d053817d57ea8220f2b07317/docs/static/api/specification.yaml
+
+package chainsync
+
+import (
+	"encoding/json"
+)
+
+// BFT Block Root
+type ByronBlockBFTV6 struct {
```

Comment:
rather than appending V6 to all our types, follow this guide for creating a new major version of the module:
https://go.dev/doc/modules/major-version

That is, create a branch called `v6` (lets stay in sync with ogmios), and just define these types as they would be normally.

Also, I wish we wouldn't have spent so much time on Byron et. al. types :sweat_smile: we'll never ever need any data from Byron. Oh well, though.